designer/client/src/i18n/translations/en.translation.json

```
@@ -41,6 +41,10 @@
       "helpText": "Use sections to split a form. For example, to add a section per applicant. The section title appears above the page title. However, if these titles are the same, the form will only show the page title.",
       "title": "Section (optional)",
       "option": "Select a section"
+    },
+    "repeatTitleField": {
+      "helpText": "Appears in the caption above the page heading",
```

Sorry, let's refer to this as "page title" which matches the field label on the **Edit page** screen

```suggestion
      "helpText": "Appears in the caption above the page title",
```